docroot/themes/humsci/su_humsci_admin/su_humsci_admin.theme

```diff
@@ -4,3 +4,14 @@
  * @file
  * su_humsci_admin.theme
  */
+
+/**
+ * Implements hook_preprocess_page().
+ */
+function su_humsci_admin_preprocess_page(&$variables) {
+  $route_name = \Drupal::routeMatch()->getRouteName();
+  if ($route_name == 'entity.node.edit_form') {
+    // We add the library here because a hook_form_alter doesn't work.
```

Comment:
Why doesn't it work? Can you add a comment?

Reply:
This sets the edit to true if the user changes anything in the ckeditor. since a user doesn't directly modify an <input> field or anything this helps with that support.
all other text fields and buttons are supported with the .on('blur', 'click' listener

Reply:
i dont know why it doesnt work. it just doesnt load 🤷‍♂️

Reply:
i can change the form, but i can't add the library for some reason

Reply:
From what I can tell with the documentation you should be able to.
https://www.drupal.org/docs/8/creating-custom-modules/adding-stylesheets-css-and-javascript-js-to-a-drupal-8-module

Reply:
Yeah i think i was just doing something dumb before. it works now in the latest commit

Reply:
I think i had it as `_form_alter($form, ....)` so the form wasn't as referenced